plugins/entity-validation/api-report.md

```diff
@@ -7,12 +7,15 @@
 
 import { BackstagePlugin } from '@backstage/core-plugin-api';
 import { JSX as JSX_2 } from 'react';
+import { ReactNode } from 'react';
 import { RouteRef } from '@backstage/core-plugin-api';
 
 // @public (undocumented)
 export const EntityValidationPage: (props: {
   defaultYaml?: string | undefined;
   defaultLocation?: string | undefined;
+  hideFileLocationField?: boolean | undefined;
+  contentHead?: ReactNode;
```

Comment:
What is this prop designed for? What's the use case?

Reply:
The contenHead is used to display some custom components above the validator. In our case, this would be a description of which fields are required and also an explanation of where these fields can be found in the catalog, mainly to help less experienced users. In addition, we would also display a note indicating that currently two entities referencing each other cannot be validated.